src/main/java/org/ehrbase/fhirbridge/ehr/converter/PatientCompositionConverter.java

```diff
@@ -0,0 +1,128 @@
+package org.ehrbase.fhirbridge.ehr.converter;
+
+import ca.uhn.fhir.rest.server.exceptions.UnprocessableEntityException;
+import com.nedap.archie.rm.archetyped.FeederAudit;
+import com.nedap.archie.rm.generic.PartySelf;
+
+import java.time.Period;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.ehrbase.fhirbridge.camel.component.ehr.composition.CompositionConverter;
+import org.ehrbase.fhirbridge.ehr.opt.geccopersonendatencomposition.GECCOPersonendatenComposition;
+import org.ehrbase.client.classgenerator.shareddefinition.Language;
+import org.ehrbase.client.classgenerator.shareddefinition.Territory;
+import org.ehrbase.client.classgenerator.shareddefinition.Setting;
+import org.ehrbase.client.classgenerator.shareddefinition.Category;
+import org.ehrbase.fhirbridge.ehr.opt.geccopersonendatencomposition.definition.AlterObservation;
+import org.ehrbase.fhirbridge.ehr.opt.geccopersonendatencomposition.definition.DatenZurGeburtCluster;
+import org.ehrbase.fhirbridge.ehr.opt.geccopersonendatencomposition.definition.EthnischerHintergrundCluster;
+import org.ehrbase.fhirbridge.ehr.opt.geccopersonendatencomposition.definition.EthnischerHintergrundDefiningCode;
+import org.ehrbase.fhirbridge.ehr.opt.geccopersonendatencomposition.definition.PersonendatenAdminEntry;
+import org.hl7.fhir.r4.model.Age;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.DateTimeType;
+import org.hl7.fhir.r4.model.Extension;
+import org.hl7.fhir.r4.model.Patient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PatientCompositionConverter implements CompositionConverter<GECCOPersonendatenComposition, Patient> {
+
+    private static final Logger LOG = LoggerFactory.getLogger(BloodPressureCompositionConverter.class);
+
+    @Override
+    public Patient fromComposition(GECCOPersonendatenComposition composition) {
+        // TODO: Implement
+        return null;
+    }
+
+    @Override
+    public GECCOPersonendatenComposition toComposition(Patient fhir_patient) {
```

Comment:
Method too long, also a lot of comments are still there, do we really need them ?

Reply:
Removed some comments and moved setting language etc. to private method.